src/components/NurseryWithdrawals/NurseryPlantingsAndWithdrawals.tsx

```diff
@@ -47,6 +47,11 @@ export default function NurseryPlantingsAndWithdrawals(): JSX.Element {
     setActiveTab(tab);
   }, [tab]);
 
+  const refreshPlantings = () => {
+    dispatch(requestPlantings(selectedOrganization.id));
```

Comment:
i think you only need to refresh listing of planting sites (not plantings or planting site search results), `plantingCompleted` is available in the subzone payload from list planting sites.
the root parent App.tsx has a refreshTracking function that we can pass into this component somehow.. take a look?

Reply:
That worked! Thanks!!

Reply:
cool!